CHANGES.md

```diff
@@ -7,6 +7,10 @@
 - Move Docker to build in 3.11 `PR #1341`
 - Add "--force-check" parameter to runner.py `PR #1347`
 
+## Bug Fixes
+
+- Fixed import of packages from requirement files with UTF-16 encoding `PR #1387`
```

Comment:
```suggestion
- Fixed reading requirement files with UTF-16 encoding supporting all encoding pip does `PR #1387`
```
We kind of do more than just UTF-16 now ... But I can fix later if you don't get to it.